staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -290,9 +292,9 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	KMSv1: {Default: true, PreRelease: featuregate.Deprecated},
 
-	KMSv2: {Default: true, PreRelease: featuregate.Beta},
+	KMSv2: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.31
 
-	KMSv2KDF: {Default: true, PreRelease: featuregate.Beta}, // lock to true in 1.29 once KMSv2 is GA, remove in 1.31
+	KMSv2KDF: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.31
```

Comment:
@liggitt @deads2k @mikedanese how do we feel about `KMSv2KDF` being locked to `true`?

Reply:
sgtm, that was the plan with v2 graduation